statistics-production/ees.qmd

```diff
@@ -281,9 +296,11 @@ The process for adding external users as collaborating analysts is similar to ad
 
 ---
 
-### What to include in an external access request
+#### What to include in an external access request
 
-If you have external users you'd like to request access for, please send the following details to the [Explore Statistics mailbox](mailto:explore.statistics@education.gov.uk), **at least two weeks** in advance of requiring access, but ideally as early as possible. Due to the dependency on DfE's digital security, we cannot guarantee access or how long it may take.
+---
+
+If you have external users you'd like to request access for, please send the following details to the [explore education statistics platforms team](mailto:explore.statistics@education.gov.uk), **at least two weeks** in advance of requiring access, but ideally as early as possible. Due to the dependency on DfE's digital security, we cannot guarantee access or how long it may take.
```

Comment:
Could the two week thing be in a callout?

Reply:
Yep, added!